sdk/servicebus/service-bus/src/receivers/sessionReceiver.ts

```diff
@@ -139,12 +147,13 @@ export class SessionReceiverImpl<ReceivedMessageT extends ReceivedMessage | Rece
   constructor(
     context: ClientEntityContext,
     public receiveMode: "peekLock" | "receiveAndDelete",
-    private _sessionOptions: SessionReceiverOptions
+    private _sessionOptions: SessionReceiverOptions,
+    receiverOptions: GetReceiverOptions
```

Comment:
Why is `receiverOptions` separate from `_sessionOptions`?

Reply:
Good point.. Fixed this by merging SessionReceiverOptions into GetSessionReceiverOptions

7c319f6